### DISCUSSION TITLE
Is there any way to write pyramid app with more typing?

### DISCUSSION THREAD

**Discussion by eisenhaus335** [Pylons/pyramid#3755] [General]
I just learned pyramid because I like a lot how it handles architecture and configuration. But i have difficulty with how it has less typing than another frameworks. Take example which Sanic and Litestar have. Their framework full of typing somewhat give confident. Pyramid although well-written architecture seems give me some hope. I could read some code and make easy guess how the code works. You can look how the pyramid project structures and take guess the typing and variable it has

Although, i still try live without less typing. But i think it be nice if pyramid typing could be improved. I still like Pyramid more than Litestar cause litestar is configuration typing hell it just tiresome to use with

**Comment by stevepiercy** [comment]
See https://github.com/Pylons/pyramid/issues/2638

**Reply by eisenhaus335** [reply]
I have did see the post. I cannot seems understand how it used on Pyramid. No example yet until now that explain how it could improve structure projects
